Azure_Pipelines/azure-pipelines.db-schema-creation.yaml

```
@@ -0,0 +1,368 @@
+  - job: getChangedFiles
+    displayName: Get Changes
+    steps: 
+      - checkout: self
+        fetchDepth: 0
+      - task: PowerShell@2
+        name: setVariables
+        inputs:
+          targetType: 'inline'
+          script: |
+            #Get any changed files
+            $changedFiles = git diff --name-status HEAD HEAD^ 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)